scripts/scil_smooth_surface.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to smooth surface from a Laplacian blur.
+"""
+
+import argparse
+import logging
+
+import numpy as np
+from trimeshpy.io import load_mesh_from_file
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+EPILOG = """
+References:
+[1] St-Onge, E., Daducci, A., Girard, G. and Descoteaux, M. 2018.
+    Surface-enhanced tractography (SET). NeuroImage.
+"""
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('surface',
+                   help='Input surface (FreeSurfer or supported by VTK).')
+
+    p.add_argument('out_surface',
+                   help='Output smoothed surface (formats supported by VTK).')
+
+    p.add_argument('-m', '--vts_mask',
+                   help='Vertices mask, where to apply the flow (.npy).')
+
+    p.add_argument('-n', '--nb_steps', type=int, default=2,
+                   help='Number of steps for laplacian smooth [%(default)s].')
+
+    p.add_argument('-s', '--step_size', type=float, default=5.0,
```

Comment:
Is this in mm ? This script is likely made for RASMM or LPSMM for human so I assume it is, but I would mention it